numpy/add_newdocs.py

```diff
@@ -1591,56 +1593,53 @@ def luf(lamdaexpr, *args, **kwargs):
     Returns
     -------
     out : ndarray or tuple of ndarrays
-        If both `x` and `y` are specified, the output array contains
-        elements of `x` where `condition` is True, and elements from
-        `y` elsewhere.
-
-        If only `condition` is given, return the tuple
-        ``condition.nonzero()``, the indices where `condition` is True.
+        The output array contains elements from `x` where `condition` is True,
+        and elements from `y` elsewhere.
 
     See Also
     --------
     nonzero, choose
 
     Notes
     -----
-    If `x` and `y` are given and input arrays are 1-D, `where` is
-    equivalent to::
+    If all the arrays are 1-D, `where` is equivalent to::
 
-        [xv if c else yv for (c,xv,yv) in zip(condition,x,y)]
+        [
```

Comment:
Nitpicky, but I usually write this as
```
[(xv if c else yv)
 for c, xv, yv in zip(condition, x, y)]
```
I feel one looses the list-comprehension aspect by moving the `[` and `]` away from the code.

Reply:
I've never seen it written like that before. I'd probably drop the parens, but the wrapping seems unobjectionable to me.